scripts/tms

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# Get the absolute path to the library
+LIB_PATH=$(python -c "
+import torch_memory_saver, importlib.machinery, pathlib, os
+package_dir = pathlib.Path(torch_memory_saver.__file__).parent
+ext_suffix = importlib.machinery.EXTENSION_SUFFIXES[0]
+lib_name = f'torch_memory_saver_cpp{ext_suffix}'
+print(str(package_dir / lib_name))
```

Comment:
nit: what about we provide such a function in torch_memory_saver python code, and here we only need to do as simple as `$(python -c "import torch_memory_saver; torch_memory_saver.that_function()"`